packages/desktop-client/src/components/accounts/TransactionsTable.test.js

```diff
@@ -393,123 +351,122 @@ describe('Transactions', () => {
     });
   });
 
-  test('keybindings enter/tab/alt should move around', () => {
+  test('keybindings enter/tab/alt should move around', async () => {
     const { container } = renderTransactions();
 
     // Enter/tab goes down/right
-    let input = editField(container, 'notes', 2);
-    fireEvent.keyDown(input, keys.ENTER);
+    let input = await editField(container, 'notes', 2);
+    await userEvent.type(input, '[Enter]');
     expectToBeEditingField(container, 'notes', 3);
 
-    input = editField(container, 'payee', 2);
-    fireEvent.keyDown(input, keys.TAB);
+    input = await editField(container, 'payee', 2);
+    await userEvent.type(input, '[Tab]');
     expectToBeEditingField(container, 'notes', 2);
 
     // Shift+enter/tab goes up/left
-    input = editField(container, 'notes', 2);
-    fireEvent.keyDown(input, keyWithShift(keys.ENTER));
+    input = await editField(container, 'notes', 2);
+    await userEvent.type(input, '{Shift>}[Enter]{/Shift}');
     expectToBeEditingField(container, 'notes', 1);
 
-    input = editField(container, 'payee', 2);
-    fireEvent.keyDown(input, keyWithShift(keys.TAB));
+    input = await editField(container, 'payee', 2);
+    await userEvent.type(input, '{Shift>}[Tab]{/Shift}');
```

Comment:
What do you think about defining an object named e.g. `keys.shiftTab` so that this strange syntax doesn’t get typo’d?

Reply:
The key codes & key syntax is pretty well documented in `user-event` docs. So I don't think we should be extracting it to our own variables.

Worst case: someone changes this. The tests will break thus not allowing people to break the syntax.